packs/feats/bullet-split.json

```diff
@@ -24,7 +24,38 @@
             "remaster": false,
             "title": "Pathfinder Guns & Gears"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "RollOption",
+                "option": "bullet-split",
+                "toggleable": true
+            },
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "bullet-split",
+                    {
+                        "or": [
+                            "item:group:firearm",
+                            "item:group:crossbow",
+                            {
+                                "and": [
```

Comment:
I'm not sure this works - only the item doing the strike has its roll options available?

Reply:
Oh, I actually misread the feat, my intent was for the penalty to apply to both types of weapons, but you only attack with the firearm/crossbow, as the flavour should've made obvious haha. I fix.